inference-engine/src/mkldnn_plugin/mkldnn_graph.h

```diff
@@ -218,6 +221,7 @@ class MKLDNNGraph {
     void AllocateWithReuse();
     void CreatePrimitives();
     void ExecuteConstantNodesOnly();
+    void SplitNodes();
```

Comment:
The method has very generic name. Maybe rename to "ExtractConstantNodes"? @maxnick what do you think?

Reply:
When you see `Extract`, you are waiting for the return value. M.b. only in my mind. What would you say about "SeparateConstantNodes"?